lib/config/duration_test.go

```diff
@@ -0,0 +1,65 @@
+package config
+
+import (
+	"bytes"
+	"encoding/json"
+	"testing"
+	"time"
+)
+
+type TestMessage struct {
+	Timeout Timeout `json:"timeout"`
+}
+
+func TestJsonDeserialization(t *testing.T) {
+	data := []byte(`{"timeout":"5m"}`)
+	var msg TestMessage
+	err := json.Unmarshal(data, &msg)
+	if err != nil {
+		t.Error(err)
+	}
+}
+
+func TestSecondDeserialization(t *testing.T) {
```

Comment:
Good point.  I added these because I wanted to make sure these various cases (second, minute, hour) worked for the end user, and writing unit tests was an easier way to verify this than  running robotest.

I've removed these two cases in b3bfe0f.